frontends/infinite-corridor/src/components/forms/EditChannelAppearanceForm.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react"
```

Comment:
*Rushed comment since I have to run:*

Would it make sense to put this component in `pages/admin`? I do not have a strong feeling.

Reply:
`forms/admin`?

Reply:
But I guess all the forms for infinite will be admin forms

Reply:
My thought was along the lines of co-locating the admin-editing code: `pages/admin` directory would contain the significant components that it uses, would import some stuff from ol-util or ol-forms. To make `pages/admin` more self-contained.

I.e., along the lines of this article Ferdi linked a while ago: https://alexmngn.medium.com/how-to-better-organize-your-react-applications-2fd3ea1920f1 *Note: If you do read that, its examples are a bit outdated. Like, people don't really have a `containers/` folder any more for redux-store connected components.*

What do you think? We can also keep it where it is for now reconsider in future prs.

Reply:
Moving to `/pages/admin`